cli/src/lib.rs

```diff
@@ -753,8 +758,9 @@ fn init(
     no_git: bool,
     jest: bool,
     template: ProgramTemplate,
+    force: bool,
 ) -> Result<()> {
-    if Config::discover(cfg_override)?.is_some() {
+    if Config::discover(cfg_override)?.is_some() && !force {
```

Comment:
Not too important but having `!force` first would be better here since we wouldn't need to run `Config::discover` if it's `force == true`.

```suggestion
    if !force && Config::discover(cfg_override)?.is_some() {
```

Reply:
Great point, thanks.